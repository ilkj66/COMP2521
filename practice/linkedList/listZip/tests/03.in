4 3 7 2 9
6 1
