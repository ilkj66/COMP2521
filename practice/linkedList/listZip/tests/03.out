Enter list 1: Enter list 2: 
List 1: [4] -> [3] -> [7] -> [2] -> [9] -> X
List 2: [6] -> [1] -> X
Zipped: [4] -> [6] -> [3] -> [1] -> [7] -> [2] -> [9] -> X
